crime-evidence/src/main/java/uk/gov/justice/laa/crime/evidence/staticdata/enums/OtherEvidenceTypes.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.laa.crime.evidence.staticdata.enums;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.stream.Stream;
+
+public enum OtherEvidenceTypes {
+    OTHER("OTHER"),
+    OTHER_BUSINESS("OTHER BUSINESS"),
+    OTHER_ADHOC("OTHER_ADHOC");
+
+    private final String evidence;
+
+    OtherEvidenceTypes(String evidence) {
+        this.evidence = evidence;
+    }
+
+    public String getEvidence() {
+        return evidence;
+    }
```

Comment:
Can we use Lombok here to cut down on the boiler plate? :)

Reply:
Updated